base_geoengine/geo_model.py

```diff
@@ -178,7 +178,8 @@ def get_edit_info_for_geo_column(self, cursor, uid, column, context=None):
         res['edit_raster'] = raster_obj.read(
             cursor, uid, raster_id[0], context=context)
         res['geo_type'] = field.geo_type
-        res['srid'] = field.srid
+        # get srid from _columns as srid is wrong in _fields
```

Comment:
Ok, on the other hand, `srid` has always the default value 900913 while `_srid` is set to the right one defined on the field.

There might be something missing in `base_geoengine/geo_field.py`

I'll have to check deeper in the code. Thus I'll put this as WIP